opba-banking-protocol-facade/src/main/java/de/adorsys/opba/protocol/facade/services/FacadeEncryptionService.java

```diff
@@ -0,0 +1,14 @@
+package de.adorsys.opba.protocol.facade.services;
+
+import de.adorsys.opba.protocol.api.services.EncryptionService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class FacadeEncryptionService {
```

Comment:
it probably is FacadeEncryptionServiceFactory

Reply:
renamed